### PR TITLE
Updates PCBuild/readme.txt

### PR DESCRIPTION
**Pull request by zooba**
Updates the PCBuild/readme.txt file for simpler install steps and new info.

### CODE REVIEW

File: PCbuild/readme.txt [hunk 1]
```diff
@@ -147,9 +135,13 @@ _msi
 _multiprocessing
 _overlapped
 _socket
-_testcapi
+_sqlite3
```
Comment (zware) [inline PCbuild/readme.txt]: This is listed below.

File: PCbuild/readme.txt [hunk 2]
```diff
@@ -15,29 +15,14 @@ Building Python using Microsoft Visual C++
 This directory is used to build CPython for Microsoft Windows NT version
 6.0 or higher (Windows Vista, Windows Server 2008, or later) on 32 and 64
 bit platforms.  Using this directory requires an installation of
-Microsoft Visual C++ 2015 (MSVC 14.0) of any edition.  The specific
```
Comment (zware) [inline PCbuild/readme.txt:18]: There's another mention of VS2015 on line 4.

**Review by zware — CHANGES_REQUESTED** [review]
Two minor things, otherwise looks good.

**Comment by zooba** [comment]
I didn't expect the Spanish Inquisition!

**Comment by zware** [comment]
I forgot to look at this again earlier, but the changes do look good.

**Pull request merged**